src/misc/flour.ts

```diff
@@ -0,0 +1,63 @@
+import { Material, Particle } from 'org.bukkit';
+import { Block } from 'org.bukkit.block';
+import { Player } from 'org.bukkit.entity';
+import { Action, BlockPlaceEvent } from 'org.bukkit.event.block';
+import { PlayerInteractEvent } from 'org.bukkit.event.player';
+import { ItemStack, PlayerInventory } from 'org.bukkit.inventory';
+import { Vector } from 'org.bukkit.util';
+
+const GRINDER_MATERIAL = Material.DARK_PRISMARINE_SLAB;
```

Comment:
Could CustomBlock API be used here? If a custom block has no persistent data, it has no overhead.